### PR TITLE
Add Run_SSIS_Package data connector wf, Knime pca service and fix FTP wf

### PR DESCRIPTION
**Pull request by aminelouati**
In this PR, we:
1. Add  Run_SSIS_Package wf in data connectors bucket
2. Add Knime pca service
3. Fix FTP wf

### CODE REVIEW

File: DataConnectors/resources/catalog/Run_SSIS_Package.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        <info name="task.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/sql-server.png"/>
+        <info name="task.documentation" value="user/ProActiveUserGuide.html#_file"/>
+      </genericInformation>
+      <scriptExecutable>
+        <script>
+          <code language="cmd">
+            <![CDATA[
+if %variables_RUNTIME_MODE% == 64-bit (
+echo Running SSIS Packages in 64-Bit 
+cd C:\Program Files\Microsoft SQL Server\150\DTS\Binn
```
Comment (fviale) [inline DataConnectors/resources/catalog/Run_SSIS_Package.xml:45]: This path should be configurable. SSIs is not necessarily installed in C:

File: Knime/METADATA.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        "name": "Knime",
+        "metadata": {
+          "kind": "Workflow/pca",
+          "commitMessage": "First commit",
+          "contentType": "application/xml"
+        },
+        "file": "resources/catalog/Knime.xml"
+      },
+      {
+        "name": "Knime_OpenRefine",
```
Comment (andrewssobral) [inline Knime/METADATA.json:27]: `"name": "Finish_Knime",`

File: DataConnectors/resources/catalog/Run_SSIS_Package.xml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+) else (
+echo Running SSIS Packages in 32-Bit
```
Comment (fviale) [inline DataConnectors/resources/catalog/Run_SSIS_Package.xml:47]: why is there no cd instruction here. Do you assume, DTExec is in the Path ? I doubt it is present by default

File: Knime/resources/catalog/Knime.xml [hunk 4]
```diff
@@ -0,0 +1,308 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.12" xsi:schemaLocation="urn:proactive:jobdescriptor:3.12 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.12/schedulerjob.xsd"  name="Knime" projectName="Cloud Automation - Deployment" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
+  <variables>
+    <variable name="INSTANCE_NAME" value="knime-instance" model=""/>
```
Comment (andrewssobral) [inline Knime/resources/catalog/Knime.xml:6]: `<variable name="INSTANCE_NAME" value="knime-instance-$PA_JOB_ID" model=""/>`

File: Knime/resources/catalog/Knime.xml [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+    <variable name="PASSWORD" value="" model="PA:Hidden"/>
+    <variable name="PROXYFIED" value="False" model="PA:Boolean"/>
+    <variable name="ENDPOINT_ID" value="knime-gui" />
```
Comment (andrewssobral) [inline Knime/resources/catalog/Knime.xml:9]: `<variable name="ENDPOINT_ID" value="knime-gui-$PA_JOB_ID" />`

File: Knime/resources/catalog/Knime.xml [hunk 6]
```diff
@@ -0,0 +1,308 @@
...
+        ss -lpn | grep -q ":$RND_PORT " || break
+    done
+    echo $RND_PORT
+}
+
+DOCKER_IMAGE=activeeon/knime-vnc
+INSTANCE_NAME=$variables_INSTANCE_NAME
+KNIME_INTERNAL_PORT=6901
+KNIME_PASSWORD=$variables_PASSWORD
+KNIME_HOST_PORT=$variables_SERVICE_PORT
```
Comment (andrewssobral) [inline Knime/resources/catalog/Knime.xml:64]: Workflow variable `SERVICE_PORT` does not exists.
It's default value should be `-1`.

**Review by andrewssobral — CHANGES_REQUESTED** [review]

File: DataConnectors/resources/catalog/Run_SSIS_Package.xml [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.12" xsi:schemaLocation="urn:proactive:jobdescriptor:3.12 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.12/schedulerjob.xsd"  name="Run_SSIS_Package" projectName="8. ETL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
+  <variables>
+    <variable name="SSIS_PACKAGE_PATH" value="/path/to/ssis_package" model="PA:NOT_EMPTY_STRING"/>
+    <variable name="SERVER" value="server_name" model="PA:NOT_EMPTY_STRING"/>
+    <variable name="EXECUTION_TOOL_PATH" value="/path/to/execution/tool" model="PA:NOT_EMPTY_STRING"/>
+  </variables>
+  <description>
+    <![CDATA[ A workflow that runs an SSIS package. Before the execution, the user has to provide the following variables:
+SSIS_PACKAGE_PATH (required) is the full path of the SSIS package deployed on the Integration Services server.
```
Comment (fviale) [inline DataConnectors/resources/catalog/Run_SSIS_Package.xml:12]: could you provide a path example in the description for both paths ?

File: DataConnectors/resources/catalog/Run_SSIS_Package.xml [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.12" xsi:schemaLocation="urn:proactive:jobdescriptor:3.12 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.12/schedulerjob.xsd"  name="Run_SSIS_Package" projectName="8. ETL" priority="normal" onTaskError="continueJobExecution"  maxNumberOfExecution="2"  >
+  <variables>
+    <variable name="SSIS_PACKAGE_PATH" value="/path/to/ssis_package" model="PA:NOT_EMPTY_STRING"/>
+    <variable name="SERVER" value="server_name" model="PA:NOT_EMPTY_STRING"/>
+    <variable name="EXECUTION_TOOL_PATH" value="/path/to/execution/tool" model="PA:NOT_EMPTY_STRING"/>
+  </variables>
+  <description>
+    <![CDATA[ A workflow that runs an SSIS package. Before the execution, the user has to provide the following variables:
+SSIS_PACKAGE_PATH (required) is the full path of the SSIS package deployed on the Integration Services server. For example, \SSISDB\Test Package\testSSIS\Package.dtsx.
```
Comment (fviale) [inline DataConnectors/resources/catalog/Run_SSIS_Package.xml:12]: is it not C:\... ?

**Review by andrewssobral — APPROVED** [review]

File: DataConnectors/resources/catalog/Run_SSIS_Package.xml (hunk 8 above)
Comment (aminelouati) [reply DataConnectors/resources/catalog/Run_SSIS_Package.xml:12]: Non the package is located inside the Integration Service server

Comment (fviale) [reply DataConnectors/resources/catalog/Run_SSIS_Package.xml:12]: ok

**Review by fviale — APPROVED** [review]

**Pull request merged**